bin/crew

```diff
@@ -1663,12 +1684,14 @@ def upload(pkg_name = nil, pkg_version = nil, gitlab_token = nil, gitlab_token_u
         end
       end
     end
-    # 6. If run architecture specfic manifests for package are missing,
-    #    attempt to install the package so the manifest files for the
-    #    currently running architecture are saved locally. (This is used
-    #    by build workflows to make sure updated manifests get
-    #    uploaded.)
-    system "crew install #{@pkg.name}", exception: false unless File.exist?("#{CREW_LOCAL_REPO_ROOT}/manifest/#{ARCH}/#{@pkg.name.chr}/#{@pkg.name}.filelist")
+    if !CREW_BUILD_NO_PACKAGE_FILE_HASH_UPDATES && !File.exist?("#{CREW_LOCAL_REPO_ROOT}/manifest/#{ARCH}/#{@pkg.name.chr}/#{@pkg.name}.filelist")
+      # 6. If run architecture specfic manifests for package are missing,
+      #    attempt to install the package so the manifest files for the
+      #    currently running architecture are saved locally. (This is used
+      #    by build workflows to make sure updated manifests get
+      #    uploaded.)
+      system "crew install #{@pkg.name}", exception: false
```

Comment:
Running crew inside of crew?  Isn't this circular redundancy?  Can we call the install function instead?